tests/e2e/puppeteer.config.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+const { executablePath } = require('puppeteer');
+
+// eslint-disable-next-line no-console, jest/require-hook
+console.log(`Using Chromium from ${executablePath()}`);
+
+module.exports = {
+	launch: {
+		devtools: process.env.PUPPETEER_DEVTOOLS === 'true',
+		headless: process.env.PUPPETEER_HEADLESS !== 'false',
+		slowMo: parseInt(process.env.PUPPETEER_SLOWMO) || 0,
+		args: [
+			'--enable-blink-features=ComputedAccessibilityInfo',
+			'--disable-web-security',
+		],
+		executablePath: executablePath(),
+	},
+};
```

Comment:
Where did you get this config from? If copied from somewhere, please include the link to the specific version of the config so it can be updated to reflect upstream changes.

Reply:
Now it will be automatically in sync with upstream - https://github.com/ampproject/amp-wp/pull/7669/commits/0f743686cbe4bef65b3caf8bf16c747b2cab57f4